frontend/src/components/FooterBar.vue

```diff
@@ -1,3 +1,205 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+import { useI18n } from 'vue-i18n';
+import { useUserStore } from '@/stores/user-store';
+
+const { t } = useI18n();
+const userStore = useUserStore();
+
+const { authenticated: isAuthenticated } = storeToRefs(userStore);
+
+const appointmentNotLoggedInNavItems = [
+  { route: 'login', i18nKey: 'logIn' },
+  { route: 'waiting-list', i18nKey: 'signUpForBeta' },
+];
+
+const appointmentLoggedInNavItems = [
+  { route: 'dashboard', i18nKey: 'dashboard' },
+  { route: 'bookings', i18nKey: 'bookings' },
+  { route: 'availability', i18nKey: 'availability' },
+  { route: 'settings', i18nKey: 'settings' },
+];
+</script>
+
 <template>
-  <div><!-- Empty for now! --></div>
+  <footer>
+    <nav class="appointment-navigation">
+      <img src="@/assets/svg/appointment_logo.svg" alt="Appointment Logo" />
+      <ul v-if="isAuthenticated">
+        <li v-for="navItem in appointmentLoggedInNavItems" :key="navItem.route">
+          <router-link :to="navItem.route">  
+            {{ t(`label.${navItem.i18nKey}`) }}
+          </router-link>
+        </li>
+      </ul>
+
+      <ul v-else>
+        <li v-for="navItem in appointmentNotLoggedInNavItems" :key="navItem.route">
+          <router-link :to="navItem.route">  
+            {{ t(`label.${navItem.i18nKey}`) }}
+          </router-link>
+        </li>
+      </ul>
+    </nav>
+
+    <hr />
+
+    <div class="mozilla-navigation">
+      <img src="@/assets/svg/mozilla-logo.svg" alt="Mozilla Logo" />
+      <div class="mozilla-navigation-content">
+        <ul>
+          <li>
+            <router-link :to="{ name: 'privacy' }" target="_blank">
+              {{ t('label.privacyPolicy' )}}
+            </router-link>
+          </li>
+          <li>
+            <a href="https://www.mozilla.org/privacy/websites/#data-tools" target="_blank">
```

Comment:
In thunderbird.net we have a handy-dandy helper function that maps keys to urls. 

Could we add something like that here? 

Add a js dict that's basically like

```
const urls = {
  'mozilla.dmca': 'https://www.mozilla.org/en-US/about/legal/report-infringement/',
  'mozilla.fraud': ...etc... 
};
```

It'll only be used for external urls, but I think it would be useful.